### ISSUE TITLE
Inquirying the details on reproducing SimplEREnv benchmark results

### ISSUE THREAD

**Issue by ledaikieu276-cmd** [2toinf/X-VLA#11]
Hi @2toinf 

I’d like to ask for some details regarding the reproduction of the experimental results on the SimplerEnv benchmark.

Could you please specify the training data used in the WidowX setting? Was it sourced from real-world data (e.g., [BridgeData V2](https://rail.eecs.berkeley.edu/datasets/bridge_release/data/) or from simulation environments such as ManiSkill?

If possible, could you also share the data preparation script and the training script, including the relevant hyperparameters (e.g., learning rate, number of epochs, etc.) used in your experiments?

Thank you very much for your time and for sharing this valuable work with the community!

Best,

**Comment by zxduan-r** [comment]
@2toinf Is it necessary to train up to 200,000 iterations? Or at what point can the training be stopped based on the loss value?

**Comment by Saberlve** [comment]
Hi @2toinf, I'm a little confused. The table in the paper indicates that 60K steps is enough for finetuning on the Bridge Dataset. Could you clarify why you trained for 200K iterations?

<img width="1280" height="404" alt="Image" src="https://github.com/user-attachments/assets/2d7586fa-3ba3-4acf-a818-410bae3437f6" />

**Comment by 2toinf** [comment]
Hey @Saberlve . Sorry for the confusion. We did observe that performance on the Simpler environment largely converges after around 60K training steps, which is why we suggest that 60K is sufficient for BridgeData in our paper. However, due to performance jitter, training for only 60K steps does not always reach the best results (e.g., ~96%) on Simpler. That’s why we encourage training a bit longer to achieve peak performance—feel free to give it a try!
